packages/react-scripts/scripts/build.js

```diff
@@ -29,7 +29,7 @@ var config = require('../config/webpack.config.prod');
 var paths = require('../config/paths');
 var checkRequiredFiles = require('react-dev-utils/checkRequiredFiles');
 var copyPublicFolder = require('react-dev-utils/paths/copyPublicFolder')(paths);
-var printFileSizes = require('react-dev-utils/paths/printFileSizes')(paths);
+var printFileSizes = require('react-dev-utils/paths/printFileSizes')(paths.appBuild);
```

Comment:
This is a bit confusing to most readers. Can you just make it a normal argument instead of a curried function?